melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/identifisering/PersonKontroller.java

```diff
@@ -14,11 +14,11 @@
     private static final int LOCAL_DATE_LENGTH = 10;
 
     static boolean harSammeStatsborgerskap(PersonModell person, SED sed) {
```

Comment:
`harFellesStatsborgerskap`, `harRegistrertStatsborgerskap` eller lignende? Ref. https://nav-it.slack.com/archives/C01CFP7SL4Q/p1614259797006200

Reply:
Jupp, skal fikse i en senere PR